src/player.ts

```diff
@@ -405,4 +405,41 @@ export class Player extends NativeInstance<PlayerConfig> {
   castStop = () => {
     PlayerModule.castStop(this.nativeId);
   };
+
+  /**
+   * Sets the playback speed of the player. Fast forward, slow motion and reverse playback are supported.
+   * @note
+   * - Slow motion is used by values between `0` and `1`.
+   * - Fast forward by values greater than `2`.
+   * - Slow reverse is used by values between `0` and `-1`, and fast reverse is used by values less than `-1`. iOS and tvOS only.
+   * @note
+   * Negative values are ignored during Casting.
+   * @note
+   * During reverse playback the playback will continue until the beginning of the active source is
+   * reached. When reaching the beginning of the source, playback will be paused and the playback
+   * speed will be reset to its default value of `1`. No {@link PlaybackFinishedEvent} will be
+   * emitted in this case.
+   *
+   * @param playbackSpeed - The playback speed level to set.
+   */
+  setPlaybackSpeed = (playbackSpeed: number) => {
+    PlayerModule.setPlaybackSpeed(this.nativeId, playbackSpeed);
+  };
+
+  /**
+   * @returns The player's current playback speed.
```

Comment:
Should we link to `setPlaybackSpeed` for details?

Reply:
Nice idea!
Address in fc50f75c65a9d3d27c6f7b935134b861538355ef